.github/workflows/ci-build.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: volta-cli/action@v1
         with:
-          node-version: 10.x
+          node-version: 12.x
```

Comment:
!!breasking change!!

Reply:
Is there a reason we're coupling this change with this PR? Do dependent packages drop support for Node 10? Can we not do this separately? As @stefanpenner mentioned, this is a breaking change, and our package.jsons' engines support still specifies Node 10.

Reply:
I found this further down the PR https://github.com/emberjs/ember-test-helpers/pull/989/files#diff-e835654135548f75db36136807171b4d12d3514e043f4f80a146b88cc668fba4R32.

I still think we need a better strategy for removing this. We could even define a new scenario for CI that uses node 10 only for embroider tests.